mcs/class/System.Web.Services/ReferenceSources/Res.cs

```diff
@@ -1,295 +1,301 @@
+		public const string WebServiceDescriptionIgnoredOptional = "WebServiceDescriptionIgnoredOptional";
+		public const string WebServiceDescriptionIgnoredRequired = "WebServiceDescriptionIgnoredRequired";
+		public const string WebServiceServer = "WebServiceServer";
+		public const string WebServiceSession = "WebServiceSession";
+		public const string WebServiceSoapVersion = "WebServiceSoapVersion";
+		public const string WebServiceUser = "WebServiceUser";
+		public const string WebShemaReferenceName = "WebShemaReferenceName";
+		public const string WebSoap11EncodingStyleNotSupported1 = "WebSoap11EncodingStyleNotSupported1";
+		public const string WebSuppressedExceptionMessage = "WebSuppressedExceptionMessage";
+		public const string WebTextMatchBadCaptureIndex = "WebTextMatchBadCaptureIndex";
+		public const string WebTextMatchBadGroupIndex = "WebTextMatchBadGroupIndex";
+		public const string WebTextMatchIgnoredTypeWarning = "WebTextMatchIgnoredTypeWarning";
+		public const string WebTextMatchMissingPattern = "WebTextMatchMissingPattern";
+		public const string WebTimeout = "WebTimeout";
+		public const string WebUnknownAttribute = "WebUnknownAttribute";
+		public const string WebUnknownAttribute2 = "WebUnknownAttribute2";
+		public const string WebUnknownAttribute3 = "WebUnknownAttribute3";
+		public const string WebUnknownElement = "WebUnknownElement";
+		public const string WebUnknownElement1 = "WebUnknownElement1";
+		public const string WebUnknownElement2 = "WebUnknownElement2";
+		public const string WebUnknownEncodingStyle = "WebUnknownEncodingStyle";
+		public const string WebUnrecognizedRequestFormat = "WebUnrecognizedRequestFormat";
+		public const string WebUnrecognizedRequestFormatUrl = "WebUnrecognizedRequestFormatUrl";
+		public const string WebVirtualDisoRoot = "WebVirtualDisoRoot";
+		public const string WebWsiContentTypeEncoding = "WebWsiContentTypeEncoding";
+		public const string WebWsiViolation = "WebWsiViolation";
+		public const string WhenUsingAMessageStyleOfParametersAsDocument0 = "WhenUsingAMessageStyleOfParametersAsDocument0";
+		public const string WireSignature = "WireSignature";
+		public const string WireSignatureEmpty = "WireSignatureEmpty";
+		public const string WsdlGenRpcLitAnonimousType = "WsdlGenRpcLitAnonimousType";
+		public const string WsdlInstanceValidationDetails = "WsdlInstanceValidationDetails";
+		public const string XmlLang = "XmlLang";
+		public const string XmlSchema = "XmlSchema";
+		public const string XmlSchemaAttributeReference = "XmlSchemaAttributeReference";
+		public const string XmlSchemaContentDef = "XmlSchemaContentDef";
+		public const string XmlSchemaElementReference = "XmlSchemaElementReference";
+		public const string XmlSchemaItem = "XmlSchemaItem";
+		public const string XmlSchemaNamedItem = "XmlSchemaNamedItem";
+		public const string indexMustBeBetweenAnd0Inclusive = "indexMustBeBetweenAnd0Inclusive";
+		public const string internalError0 = "internalError0";
```

Comment:
we should keep the previous nice text strings.

Reply:
I don't find this change anymore; maybe it was part of that revert?